### PR TITLE
Mirror Istio images

### PR DESCRIPTION
**Pull request by drpebcak**
Temporary until the work associated with #21221 is done.

This mirrors istio images to the rancher docker org.

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -77,6 +77,26 @@ steps:
...
 
+- name: docker-mirror-istio
+  image: plugins/docker
+  environment:
+    PASSWORD:
+      from_secret: docker_password
+    USERNAME:
+      from_secret: docker_username
+  commands:
+  - docker login -u $USERNAME -p $PASSWORD
```
Comment (cloudnautique) [inline .drone.yml]: I think you need to use DinD image or dapper even. The plugin entrypoint is the plugin-binary.

Comment (drpebcak) [reply .drone.yml]: we did something similar here that seems to have worked: https://github.com/rancher/ingress-nginx/blob/46c70a1f6919537b25affa642ffbec6db10292be/.drone.yml#L20

Not sure if mounting in the docker socket was required.

Comment (cloudnautique) [reply .drone.yml]: The image you are using here is plugins/docker that uses library docker
https://github.com/rancher/ingress-nginx/blob/46c70a1f6919537b25affa642ffbec6db10292be/.drone.yml#L21
so it brings the CLI uses the host docker socket.

Comment (drpebcak) [reply .drone.yml]: ah, yes. alright let me switch over to that

Comment (cloudnautique) [reply .drone.yml]: Volume mount for the host socket

Comment (cloudnautique) [reply .drone.yml]: Nevermind

**Pull request merged**